api/src/main/java/com/lohika/morning/spark/presentation/api/AttendeeAggregatorApplication.java

```diff
@@ -1,22 +1,36 @@
 package com.lohika.morning.spark.presentation.api;
 
 import com.lohika.morning.spark.presentation.spark.driver.configuration.SparkContextConfiguration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.EnableAspectJAutoProxy;
 import org.springframework.context.annotation.Import;
 
 @Configuration
 @EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class})
 @ComponentScan("com.lohika.morning.spark.presentation.api.*")
 @Import(SparkContextConfiguration.class)
-public class ApplicationConfiguration {
+public class AttendeeAggregatorApplication {
```

Comment:
Do we really need to duplicate application name in configuration? For me it is redundant =(